examples/basic/WORKSPACE

```diff
@@ -0,0 +1,13 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# ----------------------------------------------pybind---------------------------------------------------
+
+pybind_version = "2.10.4"
+
+http_archive(
+    name = "pybind11",
+    build_file = "@pybind11_bazel//:pybind11.BUILD",
+    strip_prefix = "pybind11-%s" % pybind_version,
+    urls = ["https://github.com/pybind/pybind11/archive/v%s.tar.gz" % pybind_version],
+)
```

Comment:
With the latest versions of Bazel, you can move this to the module file:
```starlark
http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
http_archive(...)
```
and you should be able to use the same version number for pybind11_bazel and pybind11.


For the record, here's the full snippet from my MODULE.bazel:

```starlark
PYBIND11_VERSION = "2.12.0"
bazel_dep(name = "pybind11_bazel", version = PYBIND11_VERSION)

http_archive = use_repo_rule("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
http_archive(
    name = "pybind11",
    build_file = "@pybind11_bazel//:pybind11-BUILD.bazel",
    strip_prefix = "pybind11-{0}".format(PYBIND11_VERSION),
    urls = ["https://github.com/pybind/pybind11/archive/v{0}.zip".format(PYBIND11_VERSION)],
    integrity = "sha256-QR93OAxDeYUGs57FlPx/K1MqE8TbZ0/PKxyjRO+u+2g=",
)
```